vars/rpmTestVersion.groovy

```diff
@@ -11,5 +11,8 @@
  */
 
 boolean call() {
+    if (params.CI_NOBUILD) {
+        return params.CI_RPM_TEST_VERSION
+    }
```

Comment:
With commit pragmas, `RPM-test-version:` implies skipping building.  Parameters should probably have the same implicit skip without people having to specify it.

Reply:
I want the RPM test version parameter to be defaulted to the close to the current release or test build version.

This makes it easy to have a separate job for running special builds or doing storage preparation on new or rebuilt CI nodes.

And having that value means that I can not just test for params.CI_RPM_TEST to have a text value.

Reply:
I think it's going to be confusing (and therefore bad) to have two different interfaces for doing the same thing that don't operate in the same way.  We already get enough of a beating for things not being intuitive enough in CI, that having two interfaces for doing the same thing operating on different assumptions will look bad on us.

Reply:
Fixed in current push.